webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -622,9 +622,12 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                 }  
             }
             if(kommunicateCommons.isObject(WIDGET_SETTINGS) && WIDGET_SETTINGS.popup) {
+                var greetingMsgTone = "0.8";
```

Comment:
@Kusum-Dhoundiyal No need for this line as you have already added default value in loadChatPopupTone function.